### PR TITLE
hc api doc

### PR DESCRIPTION
**Pull request by StekPerepolnen**
### Changelog entry 

hc api improvement

### Changelog category 

* Documentation (changelog entry is not required)

### Additional information

...

### CODE REVIEW

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 1]
```diff
@@ -38,37 +49,98 @@ message IssueLog {
...
+| **COMPUTE_POOL** ||
+| `Pool usage is over than 90/95/99%` | One of the pools' CPUs is overloaded. |
+| **NODE_UPTIME** ||
+| `Node is restarting too often/The number of node restarts has increased` | The number of node restarts has exceeded the threshold. |
+| **NODES_SYNC** ||
+| `The nodes have a time difference of ... ms` | Time drift on nodes might lead to potential issues with coordinating distributed transactions. |
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:146]: It needs some details about what level of time difference is acceptable, maybe via a link to other content.

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 2]
```diff
@@ -38,37 +49,98 @@ message IssueLog {
...
+| **LOAD_AVERAGE** ||
+| `LoadAverage above 100%` | A physical host is overloaded. </br> The `Healthcheck` tool monitors system load by evaluating the current workload in terms of running and waiting processes (load) and comparing it to the total number of logical cores on the host (cores). For example, if a system has 8 logical cores and the current load value is 16, the load is considered to be 200%. </br> `Healthcheck` only checks if the load exceeds the number of cores (load > cores) and reports based on this condition. This indicates that the system is working at or beyond its capacity, potentially due to a high number of processes waiting for I/O operations. </br></br> Load Information: </br> Source: </br>`/proc/loadavg` </br> Logical Cores Information </br></br>The number of logical cores: </br>Primary Source: </br>`/sys/fs/cgroup/cpu.max` </br></br>Fallback Source: </br>`/sys/fs/cgroup/cpu/cpu.cfs_quota_us` </br> `/sys/fs/cgroup/cpu/cpu.cfs_period_us` </br>The number of cores is calculated by dividing the quota by the period (quota / period)
+| **COMPUTE_POOL** ||
+| `Pool usage is over than 90/95/99%` | One of the pools' CPUs is overloaded. |
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:142]: This explanation doesn't sound correct:

1. A CPU is the whole processor, not a core. Are we indeed talking about cores or maybe threads?
2. If a pool has many threads, why would it say it's 90%+ used if only one is overloaded?

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 3]
```diff
@@ -38,37 +49,98 @@ message IssueLog {
...
+| `System tablet BSC didn't provide information` | Storage diagnostics will be generated with alternative way. |
+| `Storage usage over 75%/85%/90%` | Need to increase disk space. |
+| **STORAGE_POOL** ||
+| `Pool degraded/has no redundancy/failed` | These issues depend solely on the underlying `STORAGE_GROUP` layer. |
+| **STORAGE_GROUP** ||
+| `Group has no vslots` ||
+| `Group degraded` | The number of disks allowed in the group is not available. |
+| `Group has no redundancy` | A storage group lost its redundancy. |
+| `Group failed` | A storage group lost its integrity. |
+||`HealthCheck` checks various parameters (fault tolerance mode, number of failed disks, disk status, etc.) and, depending on this, sets the appropriate status and displays a message. |
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:110]: This row seems out of place as the first column is empty; what it is supposed to describe?

**Review by blinkov — CHANGES_REQUESTED** [review]

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md (hunk 3 above)
Comment (StekPerepolnen) [reply ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:110]: It describes `Group degraded`, `Group has no redundancy`, `Group failed` in common.

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md (hunk 2 above)
Comment (StekPerepolnen) [reply ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:130]: 1. A CPU is the whole processor, not a core. Are we indeed talking about cores or maybe threads?
In this context, we are talking about threads. CPUs have physical cores, and each core can run multiple threads (logical cores).

2. If a pool has many threads, why would it say it's 90%+ used if only one is overloaded?
If only one thread is overloaded but the pool shows 90%+ usage, it means the work is not evenly spread out. One thread is doing most of the work, making the overall usage look high.

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 4]
```diff
@@ -38,37 +49,98 @@ message IssueLog {
...
-* `Available size is less than 12%/9%/6%`: Free space on the physical disk is running out.
-* `VDisk is not available`: A virtual disk is not available.
-* `VDisk state is ...`: Indicates problems with a virtual disk.
-* `DiskSpace is ...`: Indicates problems with virtual disk space.
-* `Storage node is not available`: A node with disks is not available.
-* `Replication in progress`: Disk replication is in progress.
-* `Group has no redundancy`: A storage group lost its redundancy.
-* `Group failed`: A storage group lost its integrity.
-* `Group degraded`: The number of disks allowed in the group is not available.
+| Message | Description |
```
[the remark this answers is not in the document]
Comment (StekPerepolnen) [reply ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:82]: It is currently impossible to separate the information into a separate solution column. I would suggest waiting for the article about setting up monitoring (using the prometheus format) and perhaps returning to this issue.

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 5]
```diff
@@ -38,37 +49,98 @@ message IssueLog {
...
+|:----|:----|
+| **DATABASE** ||
+| `Database has multiple issues`</br>`Database has compute issues`</br>`Database has storage issues` | These issues depend solely on the underlying `COMPUTE` and `STORAGE` layers. This is the most general status of the database. |
+| **STORAGE** ||
+| `There are no storage pools` | Unable to determine `STORAGE_POOLS` issues below. |
+| `Storage degraded`</br>`Storage has no redundancy`</br>`Storage failed` | These issues depend solely on the underlying `STORAGE_POOLS` layer. |
+| `System tablet BSC didn't provide information` | Storage diagnostics will be generated with alternative way. |
+| `Storage usage over 75%/85%/90%` | Need to increase disk space. |
```
[the remark this answers is not in the document]
Comment (StekPerepolnen) [reply ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:90]: I suggest postponing specific decisions and, perhaps, returning to this issue after the documentation on setting up monitoring

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 6]
```diff
@@ -7,9 +7,31 @@ description: "The article will tell you how to initiate the check using the Heal
 
 {{ ydb-short-name }} has a built-in self-diagnostic system, which can be used to get a brief report on the database status and information about existing problems.
 
-To initiate the check, call the `SelfCheck` method from `Ydb.Monitoring`. You must also pass the name of the checked DB as usual.
+To initiate the check, call the `SelfCheck` method from SDK `Ydb.Monitoring`. You must also pass the name of the checked DB as usual.
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:10]: it sounds like `Ydb.Monitoring` is a class name from some other programming language, while example below uses `TMonitoringClient`

```suggestion
To initiate the check, call the `SelfCheck` method from a monitoring client bundled with {{ ydb-short-name }} SDK. You must also pass the name of the checked DB as usual. Examples:
```

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 7]
```diff
@@ -7,9 +7,31 @@ description: "The article will tell you how to initiate the check using the Heal
...
+
+  ```
+  auto settings = TSelfCheckSettings();
+  settings.ReturnVerboseStatus(true);
+  auto result = client.SelfCheck(settings).GetValueSync();
+  ```
+
+{% endlist %}
+
+## Response Structure {#response-structure}
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:31]:
```suggestion
## Response structure {#response-structure}
```

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 8]
```diff
@@ -38,37 +52,564 @@ message IssueLog {
...
+
+Description of all fields in the response is provided below:
+
+| Field | Description |
+|:----|:----|
+| `self_check_result` | enum field which contains the DB check result:<ul><li>`GOOD`: No problems were detected.</li><li>`DEGRADED`: Degradation of one of the database systems was detected, but the database is still functioning (for example, allowable disk loss).</li><li>`MAINTENANCE_REQUIRED`: Significant degradation was detected, there is a risk of availability loss, and human maintenance is required.</li><li>`EMERGENCY`: A serious problem was detected in the database, with complete or partial loss of availability.</li></ul> |
+| `issue_log` | This is a set of elements, each of which describes a problem in the system at a certain level. |
+| `issue_log.id` | A unique problem ID within this response. |
+| `issue_log.id` | A unique problem ID within this response. |
+| `issue_log.id` | A unique problem ID within this response. |
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:71]: weird duplicate rows

**Review by blinkov — DISMISSED** [review]
many comments from the previous iteration are not fixed

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 9]
```diff
@@ -38,37 +52,562 @@ message IssueLog {
...
+| `Pool usage is over than 90%` <br>`Pool usage is over than 95%` <br>`Pool usage is over than 99%` | One of the pools' CPUs is overloaded. |
+| **NODE_UPTIME** ||
+| `The number of node restarts has increased` | The number of node restarts has exceeded the threshold. By default, 10 restarts per hour |
+| `Node is restarting too often` | The number of node restarts has exceeded the threshold. By default, 30 restarts per hour |
+| **NODES_TIME_DIFFERENCE** ||
+| `The nodes have a time difference of ... ms` | Time drift on nodes might lead to potential issues with coordinating distributed transactions. This issus starts to appear from 5 ms |
+
+
+## Example {#examples}
+The shortest `HealthCheck` response looks like this. It is returned if there is nothing wrong with the database
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:161]:
```suggestion
## Example {#examples}

The shortest `HealthCheck` response looks like this. It is returned if there is nothing wrong with the database
```

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (blinkov) [reply ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:105]: How is this related? The guide on setting up monitoring will be about configuring the infrastructure, not fixing the issues from this table.

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 10]
```diff
@@ -38,37 +49,98 @@ message IssueLog {
...
+| `issue_log` | This is a set of elements, each of which describes a problem in the system at a certain level. |
+| `issue_log.id` | A unique problem ID within this response. |
+| `issue_log.status` | Status (severity) of the current problem. <br/>It can take one of the following values:</li><li>`RED`: A component is faulty or unavailable.</li><li>`ORANGE`: A serious problem, we are one step away from losing availability. Maintenance may be required.</li><li>`YELLOW`: A minor problem, no risks to availability. We recommend you continue monitoring the problem.</li><li>`BLUE`: Temporary minor degradation that does not affect database availability. The system is expected to switch to `GREEN`.</li><li>`GREEN`: No problems were detected.</li><li>`GREY`: Failed to determine the status (a problem with the self-diagnostic mechanism).</li></ul> |
+| `issue_log.message` | Text that describes the problem. |
+| `issue_log.location` | Location of the problem. |
+| `issue_log.reason` | This is a set of elements, each of which describes a problem in the system at a certain level. |
+| `issue_log.type` | Problem category (by subsystem). |
```
[the remark this answers is not in the document]
Comment (blinkov) [reply ydb/docs/en/core/reference/ydb-sdk/health-check-api.md]: It doesn't seem to be fixed

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 11]
```diff
@@ -38,37 +49,98 @@ message IssueLog {
...
+| `issue_log.level` | Depth of the problem nesting. |
+| `database_status` | If settings contains `verbose` parameter than `database_status` field will be filled. <br/>It provides a summary of the overall health of the database. <br/>It's used to quickly review the overall health of the database, helping to assess its health and whether there are any serious problems at a high level. |
```
[the remark this answers is not in the document]
Comment (blinkov) [reply ydb/docs/en/core/reference/ydb-sdk/health-check-api.md]: An example helps, but let's explain it right here

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 12]
```diff
@@ -38,37 +49,98 @@ message IssueLog {
...
+| `issue_log` | This is a set of elements, each of which describes a problem in the system at a certain level. |
+| `issue_log.id` | A unique problem ID within this response. |
+| `issue_log.status` | Status (severity) of the current problem. <br/>It can take one of the following values:</li><li>`RED`: A component is faulty or unavailable.</li><li>`ORANGE`: A serious problem, we are one step away from losing availability. Maintenance may be required.</li><li>`YELLOW`: A minor problem, no risks to availability. We recommend you continue monitoring the problem.</li><li>`BLUE`: Temporary minor degradation that does not affect database availability. The system is expected to switch to `GREEN`.</li><li>`GREEN`: No problems were detected.</li><li>`GREY`: Failed to determine the status (a problem with the self-diagnostic mechanism).</li></ul> |
+| `issue_log.message` | Text that describes the problem. |
+| `issue_log.location` | Location of the problem. |
```
[the remark this answers is not in the document]
Comment (blinkov) [reply ydb/docs/en/core/reference/ydb-sdk/health-check-api.md]: Bow it says "information about host", which is better, but still not specific. Is it a hostname or something else?

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 13]
```diff
@@ -38,37 +49,98 @@ message IssueLog {
...
+| `issue_log.reason` | This is a set of elements, each of which describes a problem in the system at a certain level. |
+| `issue_log.type` | Problem category (by subsystem). |
+| `issue_log.level` | Depth of the problem nesting. |
+| `database_status` | If settings contains `verbose` parameter than `database_status` field will be filled. <br/>It provides a summary of the overall health of the database. <br/>It's used to quickly review the overall health of the database, helping to assess its health and whether there are any serious problems at a high level. |
+| `location` | Contains information about host, where `HealthCheck` service was called |
+
+
+## Call parameters {#call-parameters}
+The whole list of extra parameters presented below:
```
[the remark this answers is not in the document]
Comment (blinkov) [reply ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:81]: not fixed

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 14]
```diff
@@ -38,37 +52,562 @@ message IssueLog {
...
+      FLUENT_SETTING_OPTIONAL(ui32, MaximumLevel);
+  };
+  ```
+
+{% endlist %}
+
+| Parameter | Type | Description |
+|:----|:----|:----|
+| `ReturnVerboseStatus` | `bool`         | As mentioned earlier, this parameter affects the filling of the `database_status` field. Default is false. |
+| `MinimumStatus`       | `EStatusFlag`  | The minimum severity status that will appear in the response. Less severe issues will be discarded. By default, all issues will be listed. |
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:100]: Let's add a link to the list of possible `EStatusFlag` values

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 15]
```diff
@@ -38,37 +52,562 @@ message IssueLog {
...
+| `MaximumLevel`        | `int32`        | The maximum depth of issues in the response. Issues at deeper levels will be discarded. By default, all issues will be listed. |
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:101]: why is it signed? what do negative values mean?

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 16]
```diff
@@ -38,37 +49,98 @@ message IssueLog {
...
+| `issue_log.message` | Text that describes the problem. |
+| `issue_log.location` | Location of the problem. |
+| `issue_log.reason` | This is a set of elements, each of which describes a problem in the system at a certain level. |
+| `issue_log.type` | Problem category (by subsystem). |
+| `issue_log.level` | Depth of the problem nesting. |
+| `database_status` | If settings contains `verbose` parameter than `database_status` field will be filled. <br/>It provides a summary of the overall health of the database. <br/>It's used to quickly review the overall health of the database, helping to assess its health and whether there are any serious problems at a high level. |
+| `location` | Contains information about host, where `HealthCheck` service was called |
+
+
+## Call parameters {#call-parameters}
```
[the remark this answers is not in the document]
Comment (blinkov) [reply ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:79]: not fixed

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 17]
```diff
@@ -39,36 +106,708 @@ message IssueLog {
...
+
+#### Tablets are restarting too often
+
+**Description:** Tablets are restarting too often.
+**Actions:** In [YDB Embedded UI](../embedded-ui/ydb-monitoring.md), go to the `Nodes` tab. Check the `Uptime` and status of the nodes. If the `Uptime` is short, review the logs to determine the reasons for the node restarts.
+
+#### Tablets/Followers are dead
+
+**Description:** Tablets are not running (probably cannot be started).
+**Actions:** In [YDB Embedded UI](../embedded-ui/ydb-monitoring.md), go to the `Nodes` tab. Check the `Uptime` and status of the nodes. If the `Uptime` is short, review the logs to determine the reasons for the node restarts.
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:314]:
```suggestion
### COMPUTE_QUOTA

#### Paths quota usage is over than 90%, Paths quota usage is over than 99%, Paths quota exhausted, Shards quota usage is over than 90%, Shards quota usage is over than 99%, Shards quota exhausted

**Description:** Quotas are exhausted.

**Actions:** Check the number of objects (tables, topics) in the database and delete any unnecessary ones.

### SYSTEM_TABLET

#### System tablet is unresponsive, System tablet response time over 1000ms, System tablet response time over 5000ms

**Description:** The system tablet is either not responding or takes too long to respond.

**Actions:** In [Embedded UI](../embedded-ui/ydb-monitoring.md), navigate to the `Storage` tab and apply the `Nodes` filter. Check the `Uptime` and the nodes' statuses. If the `Uptime` is short, review the logs to determine the reasons for the node restarts.

### TABLET

#### Tablets are restarting too often

**Description:** Tablets are restarting too frequently.

**Actions:** In [Embedded UI](../embedded-ui/ydb-monitoring.md), navigate to the `Nodes` tab. Check the `Uptime` and the nodes' statuses. If the `Uptime` is short, review the logs to determine the reasons for the node restarts.

#### Tablets/Followers are dead

**Description:** Tablets are not running (likely cannot be started).

**Actions:** In [Embedded UI](../embedded-ui/ydb-monitoring.md), navigate to the `Nodes` tab. Check the `Uptime` and the nodes' statuses. If the `Uptime` is short, review the logs to determine the reasons for the node restarts.
```

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 18]
```diff
@@ -39,36 +106,708 @@ message IssueLog {
...
+**Description:** ([Load](https://en.wikipedia.org/wiki/Load_(computing)).) A physical host is overloaded. This indicates that the system is working at or beyond its capacity, potentially due to a high number of processes waiting for I/O operations.
+**Logic of work:**
+Load Information:
+  Source: `/proc/loadavg`
+We use the first number of the three — the average load over the last 1 minute.
+Logical Cores Information:
+  Primary Source: `/sys/fs/cgroup/cpu.max`
+  Fallback Source: `/sys/fs/cgroup/cpu/cpu.cfs_quota_us`, `/sys/fs/cgroup/cpu/cpu.cfs_period_us`.
+The number of cores is calculated by dividing the quota by the period (quota / period).
+**Actions:** Check the CPU load on the nodes.
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:329]:
```suggestion
### LOAD_AVERAGE

#### LoadAverage above 100%

### LOAD_AVERAGE

#### LoadAverage above 100%

**Description:** A physical host is overloaded, meaning the system is operating at or beyond its capacity, potentially due to a high number of processes waiting for I/O operations. For more information on load, see [Load (computing)](https://en.wikipedia.org/wiki/Load_(computing)).

**Logic of work:**

- Load Information:

  - Source: `/proc/loadavg`
  - The first number of the three represents the average load over the last 1 minute.

- Logical Cores Information:

  - Primary Source: `/sys/fs/cgroup/cpu.max`
  - Fallback Sources: `/sys/fs/cgroup/cpu/cpu.cfs_quota_us`, `/sys/fs/cgroup/cpu/cpu.cfs_period_us`
-
 The number of cores is calculated by dividing the quota by the period (quota / period).

**Actions:** Check the CPU load on the nodes.
```

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 19]
```diff
@@ -39,36 +106,708 @@ message IssueLog {
...
+### NODES_TIME_DIFFERENCE
+
+#### Node is ... ms behind peer [id], Node is ... ms ahead of peer [id]
+
+**Description:** Time drift on nodes might lead to potential issues with coordinating distributed transactions. This issue starts to appear from 5 ms.
+**Actions:** Check for discrepancies in system time between the nodes listed in the alert, and verify the operation of the time synchronization process.
+
+## Examples {#examples}
+
+The shortest `HealthCheck` response looks like this. It is returned if there is nothing wrong with the database
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:359]:
```suggestion
### COMPUTE_POOL

#### Pool usage is over than 90%, Pool usage is over than 95%, Pool usage is over than 99%

**Description:** One of the pools' CPUs is overloaded.

**Actions:** Add cores to the configuration of the actor system for the corresponding CPU pool.

### NODE_UPTIME

#### The number of node restarts has increased

**Description:** The number of node restarts has exceeded the threshold. By default, this is set to 10 restarts per hour.

**Actions:** Check the logs to determine the reasons for the process restarts.

#### Node is restarting too often

**Description:** The number of node restarts has exceeded the threshold. By default, this is set to 30 restarts per hour.

**Actions:** Check the logs to determine the reasons for the process restarts.

### NODES_TIME_DIFFERENCE

#### Node is ... ms behind peer [id], Node is ... ms ahead of peer [id]

**Description:** Time drift on nodes might lead to potential issues with coordinating distributed transactions. This issue starts to appear when the time difference is 5 ms or more.

**Actions:** Check for discrepancies in system time between the nodes listed in the alert, and verify the operation of the time synchronization process.

## Examples {#examples}

The shortest `HealthCheck` response looks like this. It is returned if there is nothing wrong with the database:

```

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 20]
```diff
@@ -39,36 +106,708 @@ message IssueLog {
...
+```json
+{
+  "self_check_result": "GOOD"
+}
+```
+
+#### Verbose example {#example-verbose}
+
+`GOOD` response with `verbose` parameter:
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:368]:
```suggestion
`GOOD` response with `verbose` parameter:

```

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 21]
```diff
@@ -39,36 +106,708 @@ message IssueLog {
...
+                ]
+            }
+        }
+    ]
+}
+```
+
+#### Emergency example {#example-emergency}
+
+Response with `EMERGENCY` status:
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:617]:
```suggestion
Response with `EMERGENCY` status:

```

**Review by fomichev3000 — APPROVED** [review]

**Pull request merged**